5Total.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<p>You technically didn't check with your platform if that is the actual size of the Data type.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>